### PR TITLE
fix: Fixes assignee and inbox name spacing issues

### PR DESCRIPTION
**Pull request by iamsivin**
# Pull Request Template

## Description
This PR includes a fix for - 
1. Assignee name label spacing issue in inbox filter.
2. Inbox name label spacing issue in conversation header.
3. Label view in RTL directions.
4. Snoozed display text spacing in the conversation header.

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?

Needed to test in direction LTR and RTL views

**Screenshots**

**Direction LTR**

image

---
image

---
image

---
image

---

**Direction RTL**

image

---
image

---
image

---
image

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: app/javascript/dashboard/assets/scss/_rtl.scss [hunk 1]
```diff
@@ -312,8 +322,16 @@
...
+  .label-wrap .label {
+    .label-color-dot {
+      margin-left: var(--space-smaller);
+      margin-right: 0;
+    }
+
+    .label-close--button {
+      margin-left: 0;
+      margin-right: var(--space-smaller);
+    }
```
Comment (scmmishra) [inline app/javascript/dashboard/assets/scss/_rtl.scss:334]: We should probably just make `.label` a `flex` and give it a gap of `--space-smaller

```suggestion
.label-wrap .label {
    display: flex;
    gap: var(--space-smaller);
}
```

LMK if this works for the use case

**Review by scmmishra — COMMENTED** [review]
@iamsivin generally see if we could do something like `flex` + `gap` combination, it automatically handles spaces instead of adding margins. This can potentially reduce the amount of special cases needed to be handled in `RTL` cases

File: app/javascript/dashboard/assets/scss/_rtl.scss (hunk 1 above)
Comment (nithindavid) [reply app/javascript/dashboard/assets/scss/_rtl.scss:334]: This looks clean 👍

Comment (iamsivin) [reply app/javascript/dashboard/assets/scss/_rtl.scss]: @scmmishra @nithindavid Updated

File: app/javascript/dashboard/assets/scss/_rtl.scss [hunk 2]
```diff
@@ -240,6 +240,11 @@
         }
       }
 
+      .assignee-label {
+        margin-left: 0;
+        margin-right: var(--space-one);
```
Comment (nithindavid) [inline app/javascript/dashboard/assets/scss/_rtl.scss:245]:
```suggestion
        margin-right: var(--space-small);
```

File: app/javascript/dashboard/components/widgets/conversation/ConversationCard.vue [hunk 3]
```diff
@@ -431,6 +431,7 @@ export default {
 
   .assignee-label {
     display: inline-flex;
+    margin-left: var(--space-one);
```
Comment (nithindavid) [inline app/javascript/dashboard/components/widgets/conversation/ConversationCard.vue:434]:
```suggestion
    margin-left: var(--space-small);
```

**Review by nithindavid — COMMENTED** [review]
Can you check the suggestions I have added and see if it's good?

**Comment by iamsivin** [comment]
> Can you check the suggestions I have added and see if it's good?

@nithindavid Those were the space between the thumbnail and user details. If we are changing this we need to change it for the rest.
image

**Review by nithindavid — APPROVED** [review]

**Pull request merged**